src/main/java/org/mockito/MockSettings.java

```diff
@@ -361,4 +361,20 @@ public interface MockSettings extends Serializable {
      * For more information and an elaborate example, see {@link Mockito#lenient()}.
      */
     MockSettings lenient();
+
+    /**
+     * Specifies strictness level for the mock.
+     * The default strictness level is STRICT_STUBS
```

Comment:
Nit: the default is the default of the runner. If you use the lenient runner, it will be lenient. I suggest to formulate it the following way:

```
The default strictness level is determined by the rule/runner used. If you are using no rule/runner, the default strictness level is LENIENT.
```

Reply:
Updated.